hardhat.config.ts

```diff
@@ -73,6 +73,13 @@ const config: HardhatUserConfig = {
       url: process.env.BSC_MAINNET_NODE || "https://bsc-dataseed.binance.org/",
       accounts: DEPLOYER_PRIVATE_KEY ? [`0x${DEPLOYER_PRIVATE_KEY}`] : [],
     },
+    sepolia: {
+      url: "https://rpc.notadegen.com/eth/sepolia",
```

Comment:
I would use a env variable. There are nodereal endpoints available for Sepolia, that we can define locally (it has an API key)